services/src/main/java/org/jboss/windup/web/services/model/AnalysisContext.java

```diff
@@ -273,6 +276,21 @@ public void removeApplication(RegisteredApplication application)
         this.getApplications().remove(application);
     }
 
+    public AnalysisContext clone()
+    {
+        AnalysisContext clone = new AnalysisContext(this.migrationProject);
```

Comment:
Yes. Sorry, I think I used the old hibernate term there as it had a "session.evict(object)" API instead of entityManager.detach(obj). :)